src/udf/wit_example/create.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+
+path=$(dirname "$0")
+cd "$path"
+
+if [ ! -f "./my_udf.component.wasm" ]; then
+    echo "my_udf.component.wasm not found, please run ./build.sh first"
+    exit 1
+fi
+
+echo "size of wasm: $(stat -f "%z" my_udf.component.wasm) bytes"
+encoded=$(base64 -i my_udf.component.wasm)
+echo "size of encoded wasm: ${#encoded} bytes"
+# debug:   23557258
+# release: 12457072
```

Comment:
The size looks much larger than I imagine. Is there a way to strip the binary? or is it possible to build on target `wasm32-unknown-unknown`?

Reply:
> is it possible to build on target `wasm32-unknown-unknown`?

This is not possible and there's a long story. But I forget the details now. 🤡

> Is there a way to strip the binary?

I think so. I remember there are some tools for this purpose. And IIRC currently debuginfo is also included.

Reply:
Please tell me when you recall the story. 🤡